evap/evaluation/views.py

```diff
@@ -94,3 +102,27 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+@login_required
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
+    message = request.POST.get("message")
+    subject = "Feedback from {}".format(sender_email)
+
+    if message:
+        mail = EmailMessage(
+            from_email=sender_email,
+            subject=subject,
+            body=message,
+            to=[settings.FEEDBACK_EMAIL])
+
+        try:
+            mail.send(False)
+            logger.info(('Sent email "{}"').format(mail.message()))
```

Comment:
False is the default, you can remove it.

also, how does `str(mail.message())` look like? does it simply print the message body?

also, please make the text `Sent feedback email: "{}". and maybe with a newline?